typescript/src/sweep.ts

```diff
@@ -247,7 +243,7 @@ async function signMainUtxoInput(
 
 /**
  * Creates and sets `scriptSig` for the transaction input at the given index by
- * combining signature, signing group public key and deposit script.
```

Comment:
Could we replace the description of `blindingFactor` and use `wallet public key` instead of `signing group public key`?
  
  ``` 
  /**
   * An 8 bytes number. Must be unique for given Ethereum address, signing group
   * public key and refund public key.
   */
  blindingFactor: BigNumber
  ```

Reply:
See https://github.com/keep-network/tbtc-v2/pull/205/commits/ee7308ba81a9e34e9bc099addbc2f0e2f0a8d1f6